examples/rsp-webpack-4/src/index.js

```diff
@@ -1,11 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
-const root = ReactDOM.createRoot(
-  document.getElementById('root')
-);
-root.render(
-  <App />
-);
+if (ReactDOM.version.startsWith('18')) {
```

Comment:
Will follow up and add something to the webpack 4 example app so it actually render a page we can test. For now I've confirmed in a separate webpack 4 app that the latest nightly with the useId changes works with React 17/18